src/commands/device/rename.ts

```diff
@@ -49,11 +49,11 @@ export default class DeviceRenameCmd extends Command {
 		const balena = getBalenaSdk();
 
 		const newName =
-			params.newName ||
+			params.newName ??
```

Comment:
As with `options.X`, the question is: can this be `''` and if it can, how do we want to treat that? In this case, I believe a device name _can_ actually be `''`. If we want to block that, we should do so at the API level as well